hera_cal/datacontainer.py

```diff
@@ -84,12 +85,24 @@ def __delitem__(self, key):
         else:
             raise ValueError('only supports setting (ant1, ant2, pol) keys')
 
-  
+    def __add__(self, D):
```

Comment:
So this just grows visibilities in time? Do we want to support any other add method? Also we should we check that all the keys between D and self match otherwise this will fail.

Reply:
Also add a doc string